app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
```

Comment:
- [ ] @for-ashraf  great work on your counter methods I loved the implementation however you forgot to call the method  `update_comment_counter` as you did in other counter methods.

Reply:
@danmainah Kindly look into my reservations, please.

Reply:
This change has been done well